src/core/gosling-track-model.ts

```diff
@@ -701,34 +670,6 @@ export class GoslingTrackModel {
                 }
             }
         });
-
-        /* Merge domains of neighbor channels (e.g., x and xe) */
```

Comment:
We do not need this anymore since we only maintain a single `d3.scale` for the x-axis or y-axis.